server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/UserStepdefs.java

```diff
@@ -145,6 +145,22 @@ public void shareMailbox(String owner, String mailbox, String shareTo) throws Th
         mainStepdefs.aclProbe.addRights(mailboxPath, shareTo, rights);
     }
 
+    @Given("^\"([^\"]*)\" shares its mailbox with write right \"([^\"]*)\" with \"([^\"]*)\"$")
+    public void shareMailboxWithWriteRight(String owner, String mailbox, String shareTo) throws Throwable {
+        MailboxPath mailboxPath = MailboxPath.forUser(owner, mailbox);
+        MailboxACL.Rfc4314Rights rights = new MailboxACL.Rfc4314Rights(MailboxACL.Right.Lookup, MailboxACL.Right.Read, MailboxACL.Right.Write);
+
+        mainStepdefs.aclProbe.addRights(mailboxPath, shareTo, rights);
+    }
+
+    @Given("^\"([^\"]*)\" does not share its mailbox \"([^\"]*)\" with \"([^\"]*)\"$")
```

Comment:
This can be done with:

```
"user" shares its "shared" with "otherUser" with rights ""
```

(Also from https://github.com/linagora/james-project/pull/1053)